### PR TITLE
Disabled screenshot in the ScanActivityImpl

### PR DESCRIPTION
**Pull request by Anwera64**
I disabled the ability to screenshot when the app is recognizing the card so that it would't be so easy to steal credit card data (name, number and expiry date) using the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # CardScan
 
+[![](https://jitpack.io/v/Anwera64/cardscan-android.svg)](https://jitpack.io/#Anwera64/cardscan-android)
```
Comment (awushensky) [inline README.md]: this is not a repository I'm aware of. is this well maintained? who is keeping this up-to-date?

**Comment by Anwera64** [comment]
It was a repo I used to implement my own version of the library in an app. It's not necessary to merge that change. Sorry, must've slipped past me, it shouldn't be on the pull request.

**Comment by awushensky** [comment]
Cool. if you remove that line, I'm fine adding this change.

**Comment by Anwera64** [comment]
Sure, I'll fix it as soon as I can.

**Comment by awushensky** [comment]
Thanks, looks great.

**Pull request merged**